src/pages/iscn/_id.vue

```diff
@@ -0,0 +1,13 @@
+<script>
+export default {
+  fetch({ redirect, params, query }) {
+    // Redirect
+    // /iscn/:iscn_id  -> /in/like/iscn?iscn_id=:iscn_id
+    redirect({
+      name: 'in-like-id',
+      params: { id: 'iscn' },
+      query: { iscn_id: params.id, ...query },
+    });
+  },
+};
```

Comment:
we can actually do [optional params in route](https://nuxtjs.org/docs/features/file-system-routing/#dynamic-routes) to merge this file with `index?